src/components/Calendar/Event/EditEvent.js

```diff
@@ -117,11 +103,8 @@ export default function EditEvent() {
 
     const editUpdateButtons = (
         <React.Fragment>
-          
-    
             <UpdateCancelButton backgroundColor={'#912338'} content="Update" onClick={() => { handleEvent() }} />
             <UpdateCancelButton backgroundColor={'#C8C8C8'} content="Cancel" onClick={() => { handleEvent() }} />
```

Comment:
Why does the cancel button call handleEvent()? It should not make a post request to the database.